.env.local.example

```diff
@@ -1,4 +1,4 @@
-KEYCLOAK_SECRET=kc-secret
+KEYCLOAK_SECRET=99043b92-c337-4869-8a3c-09822db0fd6c

```

Comment:
@Marina-yoya Please undo this change as it exposes a secret that should not be part of the repo! :shield: :closed_lock_with_key: